FWCore/ParameterSet/python/pfnInPath.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+import os, os.path
+
+def pfnInPath(name):
```

Comment:
here's how complicated it is to reimplement (part of) the `FileInPath` logic in python.

Reply:
How about changing the name to `_pfnInPath` so that `import *` will not bring that name into the calling module?

Reply:
OK